### ISSUE TITLE
Ignore hidden files when performing precompile checks

### ISSUE THREAD

**Issue by dmdashenkov** [SpineEventEngine/tools#56]
For now, when building a Spine project with gradle we may get an error when hitting a hidden / not readable file.

```
* What went wrong:
Execution failed for task ':server:checkFqn'.
> Cannot read the contents of the file: /..../src/main/java/org/spine3/.DS_Store
```

Files like this should be emmited.

**Comment by alexander-aleksandrov** [comment]
PTAL [here ](https://github.com/SpineEventEngine/tools/pull/57)for changes

**Comment by dmdashenkov** [comment]
@alexander-yevsyukov. FYI, the issue is fixed in #57. Please migrate to `tools` of version `0.8.6-SNAPSHOT` to restore the compile-time javadoc style check.
